server/server.js

```
@@ -17,7 +16,7 @@ module.exports = function(port, googleAuthoriser) {
                 console.log("success");
                 adminToken = token;
                 res.cookie("sessionToken", token);
-                res.header("Location", "/dash.html");
+                res.header("Location", "/admin/dash.html");
```

Not sure dash should be on /admin, administrators need to be able to access the page before they've logged in

---

i just put it in a different folder to fix the lint problem, stephen's pull request sets it in a different format anyway so it's not gonna be in /admin

---

What was the lint problem?

---

'angular' not defined
- needed a .jshint file in that specific folder to declar angular as global

---

Cool, I can see that in Stephen's pull request. You could add it in this one, or just ignore the error, since moving the dash page is going to break functionality.